src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -236,4 +245,17 @@ private CommandResult executeCommand(String commandText) throws CommandException
             throw e;
         }
     }
+
+    /**
+     * Switch main display pane to the specified UI part
+     */
+    private void switchToTakeNoteView() {
+        personListPanelPlaceholder.getChildren().clear();
+        personListPanelPlaceholder.getChildren().add(noteListPanel.getRoot());
+    }
```

Comment:
not sure which panel this will affect, but I thought any command that affect NoteEntry will only change NotePanel? aka no need to switch between view?